tests/helpers/properties.js

```diff
@@ -1,37 +1,56 @@
 import AcceptanceExecutionContext from 'ember-cli-page-object/-private/execution_context/acceptance';
 import IntegrationExecutionContext from 'ember-cli-page-object/-private/execution_context/integration';
 import { AcceptanceAdapter, moduleForAcceptance, testForAcceptance } from './properties/acceptance-adapter';
+
 import { IntegrationAdapter, moduleForIntegration, testForIntegration } from './properties/integration-adapter';
 
+import { useNativeDOMHelpers } from 'ember-cli-page-object/extend';
+
 export function moduleForProperty(name, cbOrOptions, cb) {
   let options = cb ? cbOrOptions : {};
   cb = cb || cbOrOptions;
 
-  // Generate acceptance tests
-  moduleForAcceptance(`Acceptance mode | Property | ${name}`, {
-    beforeEach() {
-      this.adapter = new AcceptanceAdapter(AcceptanceExecutionContext);
-    },
+  [true, false].forEach(_useNativeDOMHelpers => {
```

Comment:
I'm not a big fan of this. But it gives us an ability to test `native-dom-helpers` almost for free. So we can use existing test suite a baseline.
I have few ideas on how to improve testing story, but I prefer to leave it for a separate issue/pr.